src/Symfony/Bundle/FrameworkBundle/Routing/Router.php

```diff
@@ -84,14 +84,12 @@ public function getRouteCollection(): RouteCollection
 
     public function warmUp(string $cacheDir, ?string $buildDir = null): array
     {
-        if (!$buildDir) {
-            return [];
+        if (null === $currentDir = $this->getOption('cache_dir')) {
```

Comment:
no it's not disabled if you set the router.cache_dir to null (which is the only way since the deprecation of the config option)

Reply:
>If you run cache:warmup right after your deploy (thus, on a read-only buildDir), this will either fail or you regenerate the router cache in the cache_dir, which won't be used at all.

then, $buildDir will be null so that RouterCacheWarmer won't call this warmup method

Reply:
> then, $buildDir will be null so that RouterCacheWarmer won't call this warmup method

Ah right, I got caught by the double warming layer of the Router... 👍